sql/dolphinscheduler-postgre.sql

```diff
@@ -499,7 +499,8 @@ CREATE TABLE t_ds_resources (
   pid int,
   full_name varchar(64),
   is_directory int,
-  PRIMARY KEY (id)
+  PRIMARY KEY (id),
+  CONSTRAINT t_ds_resources_un UNIQUE (full_name, type)
```

Comment:
Hi,
I add the CONSTRAINT and test in my local multithread env, it will show the error like[1], not [2]. But I think this solution LGTM.

BTW, the db sql schema changes should be add in a new dir maybe `1.3.3_schema`  in `upgrade` dir[3] that should be decide by @dailidong .

[1] ![image](https://user-images.githubusercontent.com/29545877/90583796-ab9a0b00-e203-11ea-9696-fa617105b1b2.png)
[2] ![image](https://user-images.githubusercontent.com/29545877/90583906-e13ef400-e203-11ea-84d3-3c0c9a4d0cc8.png)
[3] ![image](https://user-images.githubusercontent.com/29545877/90584020-2b27da00-e204-11ea-9226-af4917c2ff36.png)